allure-report-face/src/util/statuses.js

```diff
@@ -0,0 +1,16 @@
+import _ from 'underscore';
+
+const statuses = {
+    failed: '#fd5a3e',
+    broken: '#ffd963',
+    canceled: '#ccc',
+    pending: '#d35ebe',
+    passed: '#97cc64'
+};
```

Comment:
Well, that library also should be removed. I saw some objections about that tiny charts in out chart. 

Let's remove it from this PR, so we can discuss it later.